### PR TITLE
WIP / draft - identify areas of the code to be changed to add annnotation support to faasd

### PR DESCRIPTION
**Pull request by atomic77**
## Motivation and Context

- [ ] I have raised an issue to propose this change **this is required**

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Commits:

- [ ] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [ ] My commit message has a body and describe how this was tested and why it is required.
- [ ] I have signed-off my commits with `git commit -s` for the Developer Certificate of Origin (DCO)

Code:

- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.

Docs:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by alexellis** [comment]
Hi @atomic77 would you kind scanning the contribution guide around commits and signing off? (this is not GPG signing)

Alex

File: pkg/provider/handlers/functions.go [hunk 1]
```diff
@@ -49,15 +51,19 @@ func GetFunction(client *containerd.Client, name string) (Function, error) {
 
 		containerName := c.ID()
 		labels, labelErr := c.Labels(ctx)
-		if labelErr != nil {
+                // TODO Here need to retrieve the special list of labels that are actually
```
Comment (alexellis) [inline pkg/provider/handlers/functions.go:54]: See for example: https://github.com/openfaas/faas-swarm/blob/ef7d9132e29edc8ff3962685ba772ad6e3f01bda/handlers/reader.go#L57

**Comment by alexellis** [comment]
Following-up from Slack, how is this going? Are you planning on a new PR?

**Comment by atomic77** [comment]
Sorry, I had some difficulty when I tried to rebase my code on top of 0.9.2 and ended up creating a new branch. There's a new draft PR here: https://github.com/openfaas/faasd/pull/86

**Pull request closed without merging**